playwright/tests/workspaces/multi-project/prebuiltProject/.project

```diff
@@ -1,49 +1,54 @@
 <?xml version="1.0" encoding="UTF-8"?>
-  <projectDescription>
-    <name>prebuiltProject</name>
-    <comment></comment>
-    <projects>
-    </projects>
-    <buildSpec>
-      <buildCommand>
-        <name>ch.ivyteam.ivy.designer.dataClasses.ui.ivyDataClassBuilder</name>
-        <arguments>
-        </arguments>
-      </buildCommand>
-      <buildCommand>
-        <name>ch.ivyteam.ivy.designer.process.ui.ivyWebServiceProcessClassBuilder</name>
-        <arguments>
-        </arguments>
-      </buildCommand>
-      <buildCommand>
-        <name>org.eclipse.jdt.core.javabuilder</name>
-        <arguments>
-        </arguments>
-      </buildCommand>
-      <buildCommand>
-        <name>org.eclipse.wst.common.project.facet.core.builder</name>
-        <arguments>
-        </arguments>
-      </buildCommand>
-      <buildCommand>
-        <name>ch.ivyteam.ivy.designer.ide.ivyModelValidationBuilder</name>
-        <arguments>
-        </arguments>
-      </buildCommand>
-      <buildCommand>
-        <name>org.eclipse.m2e.core.maven2Builder</name>
-        <arguments>
-        </arguments>
-      </buildCommand>
-    </buildSpec>
-    <natures>
-      <nature>ch.ivyteam.ivy.project.IvyProjectNature</nature>
-      <nature>org.eclipse.wst.common.modulecore.ModuleCoreNature</nature>
-      <nature>org.eclipse.jem.workbench.JavaEMFNature</nature>
-      <nature>org.eclipse.jdt.core.javanature</nature>
-      <nature>org.eclipse.m2e.core.maven2Nature</nature>
-      <nature>org.eclipse.jem.beaninfo.BeanInfoNature</nature>
-      <nature>org.eclipse.wst.common.project.facet.core.nature</nature>
-      <nature>org.eclipse.wst.jsdt.core.jsNature</nature>
-    </natures>
-  </projectDescription>
+<projectDescription>
+	<name>prebuiltProject</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>ch.ivyteam.ivy.designer.dataClasses.ui.ivyDataClassBuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+		<buildCommand>
+			<name>ch.ivyteam.ivy.designer.process.ui.ivyWebServiceProcessClassBuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+		<buildCommand>
+			<name>org.eclipse.jdt.core.javabuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+		<buildCommand>
+			<name>org.eclipse.wst.common.project.facet.core.builder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+		<buildCommand>
+			<name>ch.ivyteam.ivy.dialog.form.build.ivyDialogFormBuilder</name>
```

Comment:
added this

Reply:
good to add this manually for now.

to keep ivy-projects under test up-2-date we maintain a central build-infrastructure, 
see https://jenkins.ivyteam.io/job/github-repo-manager_raise-ivy-projects-version/job/master/
to participate in these auto-migrations, please add vscode-extensions to the build: 
https://github.com/axonivy/github-repo-manager/blob/master/build/raise-ivy-projects-version/raise.sh#L74